<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>310</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>407</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>407</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/options.png</normaloff>icons/options.png</iconset>
  </property>
  <widget class="QTabWidget" name="OptionTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>261</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>391</width>
     <height>261</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>391</width>
     <height>261</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="UserData">
    <attribute name="icon">
     <iconset>
      <normaloff>icons/user.png</normaloff>icons/user.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Users info</string>
    </attribute>
    <widget class="QLineEdit" name="LoginEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="PasswordEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="LoginLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Login:</string>
     </property>
    </widget>
    <widget class="QLabel" name="PasswordLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UpdateData">
    <attribute name="icon">
     <iconset>
      <normaloff>icons/other.png</normaloff>icons/other.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Other</string>
    </attribute>
    <widget class="QLineEdit" name="IpLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>301</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>301</width>
       <height>20</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="IpLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IP address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="UpdateLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Update per</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="UpdateSpinBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>91</width>
       <height>22</height>
      </size>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
    <widget class="QLabel" name="UpdateLabelSec">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>seconds</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
